src/lib/filter/adjudicators.ts

```diff
@@ -93,25 +120,34 @@ export const definesNamespaces = pipe(definedNamespaces, equals([]), not);
 export const definedNamespaceRegexes = pipe(binding => binding?.filters?.regexNamespaces, defaultTo([]));
 export const definesNamespaceRegexes = pipe(definedNamespaceRegexes, equals([]), not);
 
-export const definedAnnotations = pipe(binding => binding?.filters?.annotations, defaultTo({}));
+export const definedAnnotations = pipe((binding: Partial<Binding>) => binding?.filters?.annotations, defaultTo({}));
```

Comment:
We should use `Binding` instead of `Partial<Binding>`, but that adds extra scope and can be addressed in smaller chunks as related to #1397

Reply:
https://github.com/defenseunicorns/pepr/issues/1511 has been created